spec/services/linked_data/language_service_spec.rb

```diff
@@ -8,33 +8,74 @@
     let(:authority_language) { nil }
 
     context 'when neither user nor authority language are passed in' do
-      it 'returns default language from Qa configuration' do
-        expect(subject).to match_array [:en]
+      context 'and site wide default language is set' do
+        before do
```

Comment:
These test changes are covering the various scenarios for setting language, including the new WILDCARD.